### PR TITLE
Update virtualbox to 5.0.8

### PR DESCRIPTION
**Pull request by scottsuch**

### CODE REVIEW

File: Casks/virtualbox.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 cask :v1 => 'virtualbox' do
-  version '5.0.6-103037'   # virtualbox.rb and virtualbox-extension-pack.rb should be upgraded together
-  sha256 '612e73aafa3dcdcf99e69e8831fb526512a4e4f8d02a0f4ac9d05baee09bb452'
+  version '5.0.8-103449'   # virtualbox.rb and virtualbox-extension-pack.rb should be upgraded together
```
Comment (vitorgalvao) [inline Casks/virtualbox.rb]: You are not following the comment.

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It was merged directly as 3f6e3ad420399826e9b69572ad020d91c8d3ae06 to keep commit history cleaner. Your contribution is still credited to you.

File: Casks/virtualbox.rb (hunk 1 above)
Comment (scottsuch) [inline Casks/virtualbox.rb]: Sorry about that. It was late and I clearly missed that comment.

Comment (vitorgalvao) [inline Casks/virtualbox.rb]: No problem. On second thought I just removed it. No one ever follows these comments, for some reasons (guess our brain just tells us to ignore it).

**Pull request closed without merging**